storage/pin/pin.go

```diff
@@ -237,18 +239,24 @@ func (p *API) UnpinFiles(addr []byte, credentials string) error {
 //     1) Whether the file is a RAW file or not
 //     2) Size of the pinned file or collection
 //     3) the number of times that particular file or collection is pinned.
-func (p *API) ListPinFiles() (map[string]FileInfo, error) {
-	pinnedFiles := make(map[string]FileInfo)
+func (p *API) ListPinFiles() ([]FileInfo, error) {
```

Comment:
I think you should consider changing this method name to `ListPins`, since a certain pin can be a collection of files or manifests `Pins` is more appropriate and easier on the reader

Reply:
also `FileInfo` should probably be `PinInfo`

Reply:
Done both